Samples/Gtk3/GtkMinimal/DemoWindow.cs

```diff
@@ -13,37 +13,41 @@ public class DemoWindow : ApplicationWindow
 
         private Notebook notebook;
 
-        public DemoWindow(Application application) : base(application, "demo_window.glade")
+        public DemoWindow(Application application) : this(application, new Builder("demo_window.glade")) { }
+        private DemoWindow(Application application, Builder builder) : base(builder.GetObject("root"))
         {
+            Application = application;
+            builder.Connect(this);
+            
             // Connect Button
-            Button.Clicked += Button_Clicked;
+            Button.OnClicked += Button_Clicked;
 
             // Create Notebook
             notebook = new Notebook();
```

Comment:
@badcel can we use the object initializer (with signal indexers) here ?

Reply:
Hm, I like to see that the "classic" style is possible, too. We should probably show the possiblities which are available.

I don't like how the samples are named, but this is for another PR.